src/helpers/setApplicationInfo.ts

```diff
@@ -0,0 +1,13 @@
+import {ApplicationInfo} from "../typings/applicationInfo";
+
+interface AppInfo { applicationInfo?: ApplicationInfo }
```

Comment:
shall we call appInfo to applicationInfo to be consistent or does this provide errors with the applicationInfo import ?